concepts/comparisons/about.md

```diff
@@ -1,2 +1,154 @@
-#TODO: Add about for this concept.
+# About
 
+A comparison operator in python, also called python relational operator, compares the values of two operands and returns `True` or `False` based on whether the condition is met. The most common comparison operators are `"<"`, `">"`, `"=="`, `">="`, `"<="`, `"!="`, `"is"`
+
+```python
+>>> 7 > 5
+True
+>>> 99 < 100
+True
+>>> 4 < 4
+False
+>>> 4 <= 4
+True
+>>> 1 >= 1
+True
+>>> 5 == 5
+True
+>>> 6 != 6 # not equal to
+False
+>>> 5 != 6
+True
+```
+
+## Comparison Chaining
+
+Comparisons can be chained arbitrarily, e.g., `x < y <= z` is equivalent to `x < y` `and` `y <= z`, except that `y` is evaluated only once (but in both cases `z` is _not_ evaluated at all when `x < y` is found to be `False`).
+
+Also unlike `C`, expressions like `a < b < c` have the interpretation that is conventional in mathematics.

```

Comment:
This sentence is confusing. Does it only make sense if I know C?

Reply:
Yes, the sentence is a bit confusing. I'd suggest we try to not use _short-hands_ like "e.g.", I mean yes it is proper English, but as a non-native English speaker myself "for example" doesn't really come to mind when reading "e.g.".

Also, `line 26` is _one_ sentence. We might need to keep to a general rule of thumb of `15 words MAX per line`. Splitting up the sentence into two or three should help that out quite a bit. 

But remember, don't "dumb it down" too much, we do require the students to know some basic English and know how to do simple text-based exercises (like back in school).

@IsaacG, please when commenting on code. "This sentence is confusing" is not enough. Could you **please** give either a reason why and/or a way it could improve? I don't want the contributor feel confused, and it also might strengthen the `imposter syndrome` that is inside every coder because they might not see the same thing that you or I see. Thank you.

Also, will I will be looking at @BethanyG's comment on this part.

Reply:
I am not very sure if should keep that sentence. Perhaps we can remove it to avoid all confusion. Let me know.

Reply:
@gurupratap-matharu - I think it may be an important to keep.  Python is an exception in interpreting three-way comparison mathematically with "basic" operators. Most other languages either define a new operator (see the spaceship for C++), or they define purpose-built methods instead of operators. 

See https://en.wikipedia.org/wiki/Three-way_comparison and https://softwareengineering.stackexchange.com/questions/316969/why-do-most-mainstream-languages-not-support-x-y-z-syntax-for-3-way-boolea.  For more details.

I suggest the following re-wording for the sentence:

Unlike in the `C` programming language (among other statically-typed languages), three-way operator comparisons in Python such as `a < b < c` have a conventional mathematical interpretation and precedence.

Reply:
Would it make sense to spell out what the conventional interpretation and precedence *is*? I haven't majored in math and I haven't written C in over a decade. Wikipedia (https://en.wikipedia.org/wiki/Three-way_comparison) doesn't make it immediately obvious to me what this sentence is telling me, ie what the various interpretations and precedences are.